sourcemash/frontend/static/js/views/browse_view.js

```diff
@@ -71,10 +77,10 @@ Sourcemash.Views.BrowseView = Backbone.View.extend({
   },
 
   close: function() {
-    _.each(this.feedCardViews, function(feedCardView) {
-        feedCardView.close();
-        feedCardView.remove();
-        feedCardView.unbind();
+    _.each(this.feedTopicViews, function(feedTopicView) {
```

Comment:
You should call this.close() in the render function so we don't have duplicate views lying around.